charts/opentelemetry-collector/templates/_config.tpl

```diff
@@ -115,9 +116,89 @@ exporters:
 {{- if .Values.standaloneCollector.enabled }}
 service:
   pipelines:
+    logs:
+      exporters: [otlp]
     metrics:
       exporters: [otlp]
     traces:
       exporters: [otlp]
 {{- end }}
 {{- end }}
+
+{{- define "opentelemetry-collector.agent.containerLogsConfig" -}}
+{{- if .Values.agentCollector.containerLogs.enabled }}
+receivers:
+  filelog:
+    include: [ /var/log/pods/*/*/*.log ]
+    exclude: [ /var/log/pods/{{ .Release.Namespace }}_{{ include "opentelemetry-collector.fullname" . }}*_*/{{ .Chart.Name }}/*.log ]
```

Comment:
Please add a comment to explain what this does. I believe we need to make this warning very clear since it can be dangerous to collect own logs.

Reply:
I have added a comment in https://github.com/open-telemetry/opentelemetry-helm-charts/pull/36/commits/4df897e8729399eaa3e4fde019ca7a3b31d9a316 and more documentation in https://github.com/open-telemetry/opentelemetry-helm-charts/pull/36/commits/f005dde14bfdd2971aad69a3c0adbe3233a78a97. Please take a look @tigrannajaryan.